pkg/apis/pmemcsi/v1alpha1/deployment_types.go

```diff
@@ -135,6 +139,8 @@ const (
 	DefaultNodeResourceMemory = "250Mi" // MB
 	// DefaultDeviceMode default device manger used for deployment
 	DefaultDeviceMode = DeviceModeLVM
+	// DefaultPmemPercentage PMEM space to reseve for the driver
+	DefaultPmemPercentage = 100
```

Comment:
Should this be `DefaultPMEMPercentage`? `LVM` in `DeviceModeLVM` is also all in caps.

Reply:
Fixed.